app/Http/Controllers/Forum/TopicsController.php

```diff
@@ -102,14 +102,17 @@ public function lock($id)
     {
         $topic = Topic::withTrashed()->findOrFail($id);
 
-        priv_check('ForumModerate', $topic->forum)->ensureCan();
+        $moderationPriv = priv_check('ForumModerate', $topic->forum);
+
+        $moderationPriv->ensureCan();
+        $userCanModerate = $moderationPriv->can();
```

Comment:
wouldn't this be always `true` by this point?

Reply:
yes, it was either add one line assigning true or this... thing. Maybe should just return true for that `ensureCan` thing.